### PR TITLE
Feature/linux centos intel

### PR DESCRIPTION
**Pull request by natalie-perlin**

### CODE REVIEW

File: libs/build_libtiff.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    module load hpc-$HPC_COMPILER
+    module try-load cmake
+    module load zlib
+    module list
+    set -x
+
+    prefix="${PREFIX:-"/opt/modules"}/$compiler/$name/$version"
+    if [[ -d $prefix ]]; then
+        [[ $OVERWRITE =~ [yYtT] ]] && ( echo "WARNING: $prefix EXISTS: OVERWRITING!";$SUDO rm -rf $prefix ) \
+                                   || ( echo "WARNING: $prefix EXISTS, SKIPPING"; exit 1 )
```
Comment (kgerheiser) [inline libs/build_libtiff.sh]: Could you change this to:

```
  if [[ -d $prefix ]]; then
      if [[ $OVERWRITE =~ [yYtT] ]]; then
          echo "WARNING: $prefix EXISTS: OVERWRITING!"
          $SUDO rm -rf $prefix
      else
          echo "WARNING: $prefix EXISTS, SKIPPING"
          exit 0
      fi
  fi
```

**Review by kgerheiser — CHANGES_REQUESTED** [review]
Looks good. Could you just change the prefix overwrite block?

File: libs/build_libtiff.sh (hunk 1 above)
Comment (natalie-perlin) [reply libs/build_libtiff.sh]: updated build_libtiff.sh as requested

**Review by kgerheiser — APPROVED** [review]

**Comment by kgerheiser** [comment]
@natalie-perlin could you update your branch. I'll merge this next.

**Pull request merged**